plugins/rocks.xml.webhelp/css/xml.rocks.css

```diff
@@ -384,19 +399,17 @@ button, input, optgroup, select, textarea {
 }
 
 /* Tooltip for 'Print' button */
-.tooltip-print {
-    display: none;
+.tooltip-print{
+     display: none;
```

Comment:
Please clean-up code. Remove extra space and add space between classname and bracket.